### DISCUSSION TITLE
"Invalid settings, please edit your preferences" on landing page on first use

### DISCUSSION THREAD

**Discussion by BenLangers** [searxng/searxng#2559] [Q&A] [answered]
Using version 2023.7.14+4380fc2b in Docker
Setup in Portainer as instructed, with a volume for the /etc/searxng directory

Started without settings.yml and just 2 environment vars to set the port and listening address:
```
      - SEARXNG_PORT=8080
      - SEARXNG_BIND_ADDRESS=0.0.0.0
```
web interface is served by nginx as reverse proxy for SSL termination. Without any further configuration, the web interface shows "Invalid settings, please edit your preferences" in a red banner.
The searxng container log shows:
```
2023-07-19 11:57:54,431 ERROR:searx.webapp: Invalid value: "oscar"
Traceback (most recent call last):
  File "/usr/local/searxng/searx/webapp.py", line 484, in pre_request
    preferences.parse_dict(request.cookies)
  File "/usr/local/searxng/searx/preferences.py", line 515, in parse_dict
    self.key_value_settings[user_setting_name].parse(user_setting)
  File "/usr/local/searxng/searx/preferences.py", line 80, in parse
    self._validate_selection(data)
  File "/usr/local/searxng/searx/preferences.py", line 76, in _validate_selection
    raise ValidationException('Invalid value: "{0}"'.format(selection))
searx.preferences.ValidationException: Invalid value: "oscar"
```
I have no clue what that means. There is no reference to "oscar" in the docker-compose file or anywhere in the settings.yml that is generated on the first run.
Simply opening preferences and saving clears the notification, but it reappears on any browser where searxng is opened for the first time. 

**How To Reproduce**
fire up a searxng container without any additional configuration than port and bind address, and open the searx page.

**Expected behavior**
A clean search interface without messages or notifications

**Screenshots & Logs**
![image](https://github.com/searxng/searxng/assets/65313483/554dc2e7-1ffd-4726-b662-389a57a80195)

How can I determine which setting is misconfigured or what would need to be changed?

**Comment by return42** [comment]
In the preferences you have saved to use theme 'oscar' / the oscar theme has been removed from SearXNG for a long time ago.

To solve your issue, go to the preference page of SearXNG and reset preferences.

**Comment by BenLangers** [comment, marked as answer]
Thanks for your fast follow-up!
It was just a clean install without any prior configuration. No theme was manually selected in the config file or preferences.
Got it working by adding the use_defaults and minimizing the rest of the settings.yml to just contain a couple of overrides. It works on a new browser session without the error message now.
